### PR TITLE
Fixed vagrant build

### PR DESCRIPTION
**Pull request by paulbriton**
Here are the changes to repair vagrant build. 

It should also repair the travis build but there are some complications:
  - Travis build depend on self generated PHPUnit 4.8
  - PHPUnit 4.8 integrates phpunit-selenium using composer
  - The version included by PHPUnit 4.8 is `~1.4` 
  - @sebastianbergmann doesn't want to change the version number (see [PR](https://github.com/sebastianbergmann/phpunit/pull/2416))

What do you suggest to include those changes in PHPUnit 4.8 ?

### CODE REVIEW

**Comment by paulbriton** [comment]
It also fixes the travis build, my bad.. But changes would not be included in the PHPUnit 4.8 phar unless to build it manually. 

Do you want me to port it to the `master` ?

**Comment by giorgiosironi** [comment]
`master` is where `3.x` releases come from, not `1.x` ones. From that conversation I understand PHPUnit 4.8 is about to become unsupported, so I am afraid solutions targeting PHPUnit 4.8 are a quick fix but will raise the problem again very soon.

What is it in the build that makes it depend on 4.8? Eventually it will have to run for multiple and never PHP versions that are not supported by 4.8.
What is it that makes your projects depend on 4.8 instead?

**Comment by paulbriton** [comment]
Yes I know about `master`. I ask this in order to also fix the test suite for this line.

I don't think there is any explicit dependence on 4.8 on the build. 
As I told you, I would build my PHPUnit phar manually. So basically there is no longer an issue. I just thought that my changes needs to be present on the PHPUnit 4.8 phar.

File: Tests/Selenium2TestCase/LogTest.php [hunk 1]
```diff
@@ -52,73 +52,74 @@
...
+        'default' => '',
+        'chrome' => 'html/test_log.html 4 Uncaught TypeError: Cannot read property \'inlineError\' of null',
+        'firefox' => 'TypeError: null has no properties'
+    );
 
     public function testLogType()
     {
-	    if($this->getBrowser() == 'opera')
-		    $this->markTestSkipped('Opera is very unstable support this command');
+        $this->markTestSkipped('Unsupported command');
```
Comment (giorgiosironi) [inline Tests/Selenium2TestCase/LogTest.php]: are these tests skipped for all browsers now? Not necessarily a problem, but a more verbose message on the problems in running them in `markTestSkipped()` would be helpful if we try to make it work later

File: Tests/Selenium2TestCaseTest.php [hunk 2]
```diff
@@ -559,12 +571,17 @@ public function testThePageCanBeRefreshed()
 
     public function testLinkEventsAreGenerated()
     {
+        $this->markTestIncomplete("Waiting for new phpunit-selenium release");
```
Comment (giorgiosironi) [inline Tests/Selenium2TestCaseTest.php]: reason for leaving these as incomplete?

**Comment by giorgiosironi** [comment]
Mostly ready for merge but I added a couple of questions on some tests that are now probably unstable.

File: Tests/Selenium2TestCase/LogTest.php (hunk 1 above)
Comment (paulbriton) [inline Tests/Selenium2TestCase/LogTest.php]: I think they still might work with Chrome and IE but this command is not in the new [WebDriver specifications](https://w3c.github.io/webdriver/webdriver-spec.html) and will not be implemented by the gecko driver.

File: Tests/Selenium2TestCaseTest.php (hunk 2 above)
Comment (paulbriton) [inline Tests/Selenium2TestCaseTest.php]: If the `alertIsPresent()` method isn't in the PHPUnit phar the `waitUntil()` method always timeouts (it throws unknown method error until timeout). That's linked with the release problem that I tried to explain.

**Comment by giorgiosironi** [comment]
Thank you for your work on the build. If you want to forward-port it to `master` I'll be glad to merge it there and release both versions.

**Pull request merged**